moon/docker/rover/run_solution.bash

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+echo "Unpause simulation"
+rosservice call /gazebo/unpause_physics "{}"
+echo "wait a moment"
+sleep 5
+
+echo "Start robot solution"
+export OSGAR_LOGS=`pwd`
+cd osgar
+python3 -m osgar.record --duration 2700 moon/scout1.json --note "collect some ROS data" &
+ROBOT_PID=$!
+cd ..
+
+# get directory where this bash script lives
+samedir=$(dirname $(readlink -f "${BASH_SOURCE[0]}"))
+
+# enable ROS DEBUG output to see if messages are being dropped
+export ROSCONSOLE_CONFIG_FILE="${samedir}/rosconsole.config"
+
+# Run your solution and wait for ROS master
+# http://wiki.ros.org/roslaunch/Commandline%20Tools#line-45
+## roslaunch subt_seed x1.launch --wait &
+python ./osgar/moon/rospy_rover.py &
```

Comment:
What happens when `rosmaster` is not yet running at this point? I remember the `--wait` option being the key when using `roslaunch`. Does `rospy_rover.py` wait for the master to become available?

Reply:
Thanks, good point. For Moon this is even worse as the simulation is not ready when master is available. It loads some plugins and generates randomized words ... yes definitely this will require revision.